_integration-schemas/pendo/v1/guides.md

```diff
@@ -10,6 +10,8 @@ description: |
   The `{{ table.name }}` table contains info about the guides in your {{ integration.display_name }} account.
 
 replication-method: "Key-based Incremental"
+replication-key:
```

Comment:
It looks like the ticket I wrote that corresponds to this was written badly, but the Rep Key change is actually for the `guide_events` table, not `guides`. I recommend always checking the pull request that corresponds with our changes to make sure you're working on the right thing.

[See the PR here](https://github.com/singer-io/tap-pendo/pull/21/files#diff-c228b69d34f7069450ae23364f654671e3c0ba6502c989b4aca087d37ae622bb)

Also, `browserTime` is `browser_time` and it's an attribute in the actual table.